src/components/tabs/Tabs.vue

```diff
@@ -127,4 +149,29 @@ export default defineComponent({
     }
   }
 }
+
+dl {
+  font-size: 14px;
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  overflow: hidden;
+
+  div {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    margin: 10px 0;
+
+    dt {
+      display: flex;
+      align-items: center;
+    }
+
+    dd {
+      display: flex;
+      align-items: center;
+    }
+  }
```

Comment:
You should style based on unique class names instead of element selectors. Something like
```suggestion
.list {
  font-size: 14px;
  display: flex;
  flex-direction: column;
  width: 100%;
  overflow: hidden;

  &__item {
    display: flex;
    flex-direction: row;
    justify-content: space-between;
    margin: 10px 0;

    &__header {
      display: flex;
      align-items: center;
    }

    &__description {
      display: flex;
      align-items: center;
    }
  }
```